mediaqueries-5/Overview.bs

```diff
@@ -2634,11 +2634,11 @@ Detecting the desire for increased or decreased color contrast from elements on
 	Type: discrete
 	</pre>
 
-	The 'prefers-contrast' media feature is used to detect if the user
-	has requested the system increase or decrease the amount of contrast
-	between adjacent colors. For example, many users have difficulty
-	reading text that has a small difference in contrast to the text
-	background and would prefer a larger contrast.
+	The 'prefers-contrast' media feature is used to detect
+	if the user has requested an increase or decrease the amount of contrast used in the page,
+	expecting an adjustment to the contrast ratio between adjacent colors,
+	and to how much elements stand out visually,
+	for example by adjusting their borders.
```

Comment:
Not quite sure how the end of this sentence hooks up to the start. Seems like perhaps a change in subject? It could probably stand being broken into a separate sentence.

Reply:
Adding parenthesis to show how I meant to group these statements, and implied bits between [brackets]:

> The 'prefers-contrast' media feature is used to detect
> if the user has requested an increase or decrease in the amount of contrast used in the page,
> [due to that user] expecting an adjustment to (the contrast ratio between adjacent colors,)
> and to (how much elements stand out visually,
> for example by adjusting their borders).

Here's a different attempt:

> The 'prefers-contrast' media feature is used to detect
> if the user has requested an increase or decrease in the amount of contrast used in the page.
> This could be responded to by adjusting the contrast ratio between adjacent colors,
> or by changing how much elements stand out visually,
> for example by adjusting their borders.

Reply:
@fantasai @tabatkins I understand both, cause I wrote both :) Could the two of you agree as to which one's better, or suggest a third alternative that makes everyone happy?